src/_locales/en/messages.json

```diff
@@ -23,5 +23,8 @@
   },
   "siteNotMonetized": {
     "message": "This website is not monetized."
+  },
+  "error_connectWallet_invalidClient": {
+    "message": "Failed to connect. Did you copy the public key to the right wallet address?"
```

Comment:
Should it say "right wallet and address"?

Reply:
As is, It's the right error message. The key is bound to the wallet address, not the wallet.